tests/mdx/jsx.mdx

```diff
@@ -18,15 +18,6 @@
 
 ---
 
-<Hello>
-    test   <World />   test
-</Hello>       123
-<Hello>
-    test   <World />   test
-</Hello>       234
```

Comment:
Please don't remove it. Better add a `FIXME` comment if you don't find a way to fix this.

Reply:
Restored. Forgot to revert this change before submitting. Thanks!